packages/suite/src/middlewares/suite/analyticsMiddleware.ts

```diff
@@ -69,6 +70,9 @@ const analytics = (api: MiddlewareAPI<Dispatch, AppState>) => (next: Dispatch) =
             break;
         case DEVICE.CONNECT: {
             const { features, mode } = action.payload;
+            const acquiredDevice = state.devices.find(device => device.type === 'acquired');
+            const isBtcOnly = acquiredDevice ? isBitcoinOnly(acquiredDevice) : false; // In practice it shouldn't happen that acquiredDevice is undefined because we are in DEVICE.CONNECT action
```

Comment:
(is that right?)

Reply:
I think you want to check the device you got in the `action.payload`. Right now you are checking a fw on the first acquired device stored in a reducer. If user has connected multiple devices then you will always report on the first device in the list which may not be the device that was just connected.

Reply:
You are right. I updated the device-search approach